### ISSUE TITLE
Postinstall fails on missing simulator runtimes

### ISSUE THREAD

**Issue by bartolkaruza** [wix/Detox#1167]
#### Description
Installing the latest version of detox into a client project fails on the postinstall script if certain simulator environments are not present. This issue looked similar on the surface as https://github.com/wix/Detox/issues/1153 and https://github.com/wix/Detox/issues/1092 but it has a distinctly different error in the [detox log](https://gist.github.com/bartolkaruza/d851d0bda180f127802b8bb27e21e652):
```
<unknown>:0: error: unable to load standard library for target 'arm64-apple-ios9.0-simulator'
```

#### Steps to Reproduce
Follow [getting started steps 1 and 2 ](https://github.com/wix/Detox/blob/master/docs/Introduction.GettingStarted.md) until ```yarn add detox --dev``` for your CLI project.

It should fail on ```yarn add --dev detox``` with the same stack as the other issues mentioned above.
```
...node_modules/detox/scripts/build_framework.ios.sh
    at checkExecSyncError (child_process.js:616:11)
    at Object.execFileSync (child_process.js:634:13)
    at Object.<anonymous> (/Users/bartolkaruza/dev/proj/jean/jean/react-native/node_modules/detox/scripts/postinstall.js:2:27)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
```

I suspect the issue will only arise on newly installed Mac OS that never had older versions of Xcode, because old versions include the iOS 9 simulator runtime. Version 10.1 (10B61) is the very first Xcode version on this machine.

#### Detox, Node, Device, Xcode and macOS Versions

* Detox: 10.0.9
* React Native: 0.58.4
* Node: v10.15.0
* Device: N/A
* Xcode: 10.1 (10B61)
* macOS: 10.14.2 (18C54)

#### Device and verbose Detox logs

[detox log](https://gist.github.com/bartolkaruza/d851d0bda180f127802b8bb27e21e652)
[yarn add log](https://gist.github.com/bartolkaruza/236078bd0425bace7aff60322bbd4332)

I'm currently installing the iOS9 runtime and I'll post on this issue if it resolves it entirely. If it does, maybe an extra step should be added to the getting started or some other way of gracefully failing on missing simulator runtimes can be devised.

**Comment by LeoNatan** [comment]
https://github.com/Carthage/Carthage/issues/2613

Seems others are seeing similar issues, but not sure what is causing this.

**Comment by bartolkaruza** [comment]
This issue is resolved for me end with 10.0.10

Not sure if something in my local environment has changed or if there is actual (accidental) fix in the 10 patch. Anyway, closing :)
